@here/harp-datasource-protocol/lib/InterpolatedProperty.ts

```diff
@@ -75,11 +86,14 @@ export function isInterpolatedProperty<T>(p: any): p is InterpolatedProperty<T>
  *
  */
 export function getPropertyValue<T>(
-    property: InterpolatedProperty<T> | MaybeInterpolatedProperty<T>,
+    property: Value | Expr | InterpolatedProperty<T>,
     level: number,
     pixelToMeters: number = 1.0
 ): number {
-    if (isInterpolatedPropertyDefinition<T>(property)) {
+    if (property instanceof Expr) {
```

Comment:
At runtime, this will be `instanceof Expr`.
This is just a guard, until #697, Exprs are filtered out by `StyleSetEvaluator` and don't pop up in main thread.
It's just inpractical to add yet another subtype of technique with types possible in main thread, different than these in decoder thread.